.github/workflows/build-images-releases.yaml

```diff
@@ -17,6 +17,7 @@ permissions:
 jobs:
   build-and-push:
     timeout-minutes: 45
+    name: Build and Push Images
```

Comment:
Should this denote that these are release images to differentiate from other types of images?